src/external-api/server.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const port = 4000;
```

Comment:
Throw this guy in an env variable: `process.env.external_api_port`.

And add that env variable to the `.env.example` file.

(and to your local `.env` file to make it work, obviously)